src/ucp/wireup/address.c

```diff
@@ -229,6 +229,7 @@ ucp_address_gather_devices(ucp_worker_h worker, ucp_ep_h ep, uint64_t tl_bitmap,
                 if (ucp_ep_get_rsc_index(ep, lane) == rsc_index) {
                     dev->tl_addrs_size += !ucp_worker_unified_mode(worker);
                     dev->tl_addrs_size += iface_attr->ep_addr_len;
+                    dev->tl_addrs_size += sizeof(uint8_t); /* lane index */
```

Comment:
sizeof(lane) ?

Reply:
this is a serialization protocol and not memory allocation so it's using specific types (it's same way in all places in address pack/unpack)

Reply:
ok